apps/transport/lib/jobs/new_dataset_notifications_job.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Transport.Jobs.NewDatasetNotificationsJob do
+  @moduledoc """
+  Job in charge of sending notifications about datasets that have been added recently.
+  """
+  use Oban.Worker, max_attempts: 3
+  import Ecto.Query
+
+  @impl Oban.Worker
+  def perform(%Oban.Job{inserted_at: %DateTime{} = inserted_at}) do
+    inserted_at |> relevant_datasets() |> Transport.DataChecker.send_new_dataset_notifications()
+    :ok
+  end
+
+  def relevant_datasets(%DateTime{} = inserted_at) do
+    datetime_limit = inserted_at |> DateTime.add(-24 * 60 * 60, :second)
```

Comment:
```suggestion
    datetime_limit = inserted_at |> DateTime.add(-1 :day)
```

Maintenant possible avec Elixir 1.14 :heart: 
https://hexdocs.pm/elixir/1.14.0/DateTime.html#add/4